src/main.tsx

```diff
@@ -12,6 +12,20 @@ import './styles/fonts.css'
 import { AnimatePresence } from 'framer-motion'
 import ConfiguredRoutes from './routes'
 import { Modals } from './Modals'
+import * as Sentry from '@sentry/react'
+
+Sentry.init({
```

Comment:
Let's put this in a separate util and expose as `initSentry()` or similar pls, e.g. something akin to https://github.com/WalletConnect/cloud-app/blob/develop/sentry.client.config.js